imports/plugins/core/ui/client/components/list/list.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component, PropTypes } from "react";
```

Comment:
React is used at runtime when the JSX is transpiled to JS. ( you will receive a runtime error is React is not imported )